pyinstalive/auth.py

```diff
@@ -45,7 +45,7 @@ def onlogin_callback(api, cookie_file):
         logger.separator()
 
 
-def authenticate(username, password, force_use_login_args=False):
+def authenticate(username, password, force_use_login_args=False, proxy=None):
```

Comment:
Needs error `ValueError` handling when the proxy is not valid or otherwise not working.

Reply:
I added url format validation. when the proxy is not working, I see these errors:
[E] Could not get user info for 'laurenorlando88': 0 URLError <urlopen error Tunnel connection failed: 407 Proxy Authentication Required>

Would that be enough for an error?